ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/EvidenceMapper.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.ccd.domain.evidence.CCDEvidence;
+import uk.gov.hmcts.cmc.ccd.domain.evidence.CCDEvidenceRow;
+import uk.gov.hmcts.cmc.domain.models.evidence.Evidence;
+
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+@Component
+public class EvidenceMapper implements Mapper<CCDEvidence, Evidence> {
+
+    private final EvidenceRowMapper evidenceRowMapper;
+
+    @Autowired
+    public EvidenceMapper(EvidenceRowMapper evidenceRowMapper) {
+        this.evidenceRowMapper = evidenceRowMapper;
+    }
+
+    @Override
+    public CCDEvidence to(Evidence evidence) {
+        if (evidence == null) {
+            return null;
+        }
+        CCDEvidence.CCDEvidenceBuilder builder = CCDEvidence.builder();
```

Comment:
mind adding a blank line after the brace please?
it would make it easier to read
REC

Reply:
done